installer/install.sh

```
@@ -0,0 +1,635 @@
+    packages_to_install=()
+    for pkg in "$@"; do
+        if dpkg -s "$pkg" >/dev/null 2>&1;
+        then
+            echo "$pkg is already installed. Skipping..."
+        else
+            packages_to_install+=("$pkg")
+        fi
+    done
+
+    if [ ${#packages_to_install[@]} -gt 0 ]; then
+        echo "Installing packages: ${packages_to_install[@]}"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)